handout/advanced-components/access_child_components.md

```diff
@@ -59,7 +58,7 @@ export class App {
 ```
 [View Example](http://plnkr.co/edit/8Eak9DANedsZDHBHuLea?p=preview)
 
-As shown above, when given a class type `@ViewChild` & `@ViewChildren` select child components based on type. However, they can also be passed selector strings:
+As shown above, given a class type to @ViewChild and @ViewChildren a child component or a list of children component are selected respectively using their selector from the template. In addition both @ViewChild and @ViewChildren can be passed a selector string:
```

Comment:
Should put backticks around `@ViewChild` and `@ViewChildren` since they are code snippets